app/controllers/concerns/wizard_steps.rb

```diff
@@ -75,6 +75,8 @@ def completed_step_path
   end
 
   def step_params
+    return {} unless params.key?(step_param_key)
```

Comment:
You could switch to using `fetch` with a default param instead?

`params.fetch(step_param_key, {}).permit ....`